src/docker/Dockerfile

```diff
@@ -257,6 +257,13 @@ RUN helm plugin install https://github.com/aslafy-z/helm-git.git --version "${HE
   && asdf reshim helm \
   && rm -rf /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
 
+# Install helm-s3 plugin. Get versions from https://github.com/hypnoglow/helm-s3/releases
+ARG HELM_S3_VERSION="=0.10.0"
```

Comment:
I think the `=0.10.0` should be `0.10.0` in order to pass the spec test.

Reply:
oof. Sorry about that